content/en/borealis/quick-start/borealis-canary-get-started.md

```diff
@@ -68,16 +68,10 @@ Retrospective analysis is the starting point to creating queries so that you can
    - **Lower Limit**: The lower limit for the query. If the results fall below this value, the deployment is considered to be a failure. Set this to `0`.
    - **Query Template**:
 
-   ```sql
```

Comment:
I think the `sql` directive makes the formatting worse here

Reply:
I don't know why 'sql' is used since this is a YAML block